internal/providers/vsphere/pke/cluster.go

```diff
@@ -0,0 +1,62 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pke
+
+import (
+	"fmt"
+
+	"github.com/banzaicloud/pipeline/internal/cluster/clusterbase"
+	intPKE "github.com/banzaicloud/pipeline/internal/pke"
+)
+
+const PKEOnVsphere = "pke-on-vsphere"
```

Comment:
Can we find a  better naming concept instead of `pke-on-azure` and `pke-on-vsphere`? I think it was better when the distribution was just `pke` and there was an additional cloud field.

Reply:
It's exactly like that, distribution name is `pke` and cloud is `vshpere`. This const is only used in one place in a logger.